cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -1,29 +1,28 @@
 NAME
-    Ballerina build - Builds Ballerina module(s) and generates executable
+    Ballerina build - Build a Ballerina module(s)/file and generate executable
```

Comment:
Can't we rephrase this in a better way? This just sounds odd. Also, `build` command is not applicable for standalone files.

Reply:
Rephrased as follows:

"Ballerina build - Build Ballerina modules/file and generate executable
    outputs."